content/en/account_management/billing/rum.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Real User Monitoring billing FAQ
+kind: faq
+---
+
+#### How is a session defined?
+
+A session is a user journey on your web application. It expires after 15 minutes of inactivity.
+
+#### What data does Datadog’s RUM collect?
+
+Datadog collects all the pages visited by your end users along with the telemetry that matters: resources loading (XHRs, images, CSS files, JS scripts, etc), frontend errors and long tasks. All of this is included in the user session. Datadog charges per ten thousand sessions ingested in the Datadog Real User Monitoring (RUM) service.
+
+#### Do you handle Single Page Application?
+
+Yes, without any configuration on your side. Datadog RUM automatically track page changes so you are confident about which pages your users visit.
```

Comment:
```suggestion
Yes, without any configuration on your side. Datadog RUM automatically tracks page changes.
```

removing last part because it's about a single page application, theres only one page